tests/ffi/alias.rs

```diff
@@ -11,7 +11,10 @@ pub mod ffi {
         #[namespace = "tests"]
         type C = crate::ffi::C;
 
+        #[namespace = "tests"]
```

Comment:
Needing a namespace attribute on every alias is going to get annoying for bridges with lots of aliases, but this at least works. An improved alternative might be to provide a mapping of Rust module to C++ namespace, e.g. "crate::ffi = tests", through the cxx::bridge attribute, but that's a bigger change that I'm less confident implementing right now.

Reply:
Ok, it wasn't a bigger change than I was confident making. 👍  Improved version implemented.